plugins/woocommerce-blocks/assets/js/base/components/notice-banner/style.scss

```
@@ -150,3 +150,8 @@
 		@extend %info;
 	}
 }
+
+// To fix notice banner width for all themes.
```

```suggestion
// To fix notice banner width for all block themes.
```

The variable `--wp--style--global--wide-size` isn't available on classic themes, and this fix only works on block themes, as classic themes do not have the template files to which you applied the changes.